docs/DEVELOPMENT.md

```diff
@@ -78,6 +78,44 @@ On only 3.9 (for example):
 Please also note that you can reuse an existing environtment if you run nox with the `-r` argument (or even `-R` if you
 don't want to attempt to reinstall packages). This can significantly speed up repeat test runs.
 
+## Check code coverage
+
+To run code coverage locally, you can either use `pytest` in one of the test environments or
+run `nox -s test` with coverage arguments. We use [pytest-cov](https://pypi.org/project/pytest-cov/) for our coverage reporting.
+
+Below is an example of running a coverage report on a single test. In this case the relevent file is `src/astro/sql/operators/sql_decorator.py`
+since we are testing the postgres `transform` decorator.
+
+```shell script
+nox -R -s test -- --cov-report term --cov=src/astro/sql/operators  tests/operators/test_postgres_decorator.py
```

Comment:
Probably worth adding  `--cov-branch` here as well, to highlight in case there are implicit `else` statements within the package/module of interest